jobs/jobs_manager.py

```diff
@@ -151,28 +149,28 @@ def refresh_state_of_beam_job_run_model(
         return
 
     try:
-        get_job_request = dataflow.GetJobRequest(
+        job = dataflow.JobsV1Beta3Client().get_job(dataflow.GetJobRequest(
```

Comment:
I'm bit confused what the changes below do? Could you explain?

Reply:
Sure, this just passes the `GetJobRequest` object directly as the first argument, instead of storing it in a variable first and then passing the variable as an argument.

I just found this syntax to be cleaner, since it's obvious that there's only one thing happening (a request to `get_job()`).